### PR TITLE
CNV-18349: RN load balancer service using MetalLB

### PR DESCRIPTION
**Pull request by sjhala-ccs**
Version(s): 4.12

Issue: [CNV-18349](https://issues.redhat.com//browse/CNV-18349)

Link to docs preview: https://54300--docspreview.netlify.app/openshift-enterprise/latest/virt/virt-4-12-release-notes.html#virt-4-12-networking-new

QE review:
- [x] QE has approved this change.

### CODE REVIEW

File: virt/virt-4-12-release-notes.adoc [hunk 1]
```diff
@@ -100,9 +100,11 @@ The SVVP Certification applies to:
 === Installation
 
 
-//[id="virt-4-12-networking-new"]
-//=== Networking
+[id="virt-4-12-networking-new"]
+=== Networking
 
+//CNV-18349
+* You can now xref:../virt/virtual_machines/vm_networking/virt-creating-service-vm.adoc#virt-about-services_virt-creating-service-vm[configure a load balancing service] by using the MetalLB Operator in layer 2 mode.
```
Comment (sbulage) [inline virt/virt-4-12-release-notes.adoc:107]: IMO, you can also add link to the MetalLB Operator with this --> https://54300--docspreview.netlify.app/openshift-enterprise/latest/networking/metallb/metallb-operator-install.html#metallb-operator-install

**Review by sbulage — CHANGES_REQUESTED** [review]
Small Change, rest LGTM 👍

File: virt/virt-4-12-release-notes.adoc (hunk 1 above)
Comment (sjhala-ccs) [reply virt/virt-4-12-release-notes.adoc:107]: @sbulage The documentation that the release note links to includes links to the topics "Installing the MetalLB Operator" and "Configuring services to use MetalLB" in the _Additional resources_ section.

Comment (phoracek) [inline virt/virt-4-12-release-notes.adoc:107]:
```suggestion
* You can now xref:../virt/virtual_machines/vm_networking/virt-creating-service-vm.adoc#virt-about-services_virt-creating-service-vm[configure a load balancing service] by using the MetalLB Operator.
```

@sbulage keep me honest here, but I believe we support both modes. Our integration in CNV is not affected by the selected mode.

**Review by phoracek — CHANGES_REQUESTED** [review]

File: virt/virt-4-12-release-notes.adoc (hunk 1 above)
Comment (sbulage) [reply virt/virt-4-12-release-notes.adoc:107]: Okay, we could highlight it more specifically. Or you can add Note, mentioning that if MetalLB is not installed here you can find links in additional resources section.

Comment (sbulage) [reply virt/virt-4-12-release-notes.adoc:107]: @phoracek TBH, it doesn't matter as MetalLB will available for not only CNV only but also for OpenShift too.

Comment (sjhala-ccs) [reply virt/virt-4-12-release-notes.adoc]: @sbulage I have linked to the "Installing the MetalLB Operator" topic from the release note per your earlier suggestion. Can you please review and approve? Thanks!

**Review by sbulage — APPROVED** [review]
LGTM 👍

**Pull request merged**